### PR TITLE
feat: support globals

### LINKED ISSUE #79: Support globals?
http://eslint.org/docs/user-guide/configuring#specifying-globals

### PR DESCRIPTION
**Pull request by AndersDJohnson**
Fixes #79

### CODE REVIEW

File: lib/processor.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ function getComment(html) {
 
     html = html.slice(commentStart.length, -commentEnd.length);
 
-    if (html.trim().slice(0, prefix.length) !== prefix) {
+    if (!html.trim().match(regex)) {
```
Comment (vsemozhetbyt) [inline lib/processor.js]: Maybe `!regex.test(html.trim)` would be more efficient in a boolean context (returns boolean value, not object)?

Comment (AndersDJohnson) [inline lib/processor.js]: @vsemozhetbyt Good call, pushed update.

**Comment by AndersDJohnson** [comment]
@vsemozhetbyt I made the requested change, does this look good now?

**Comment by vsemozhetbyt** [comment]
@AndersDJohnson Yes, but I am not a collaborator in this repository and have not any approval rights)

**Comment by vsemozhetbyt** [comment]
If I get it right, this repository is updated seldom, so be not discouraged with a long review delay.

**Comment by btmills** [comment]
> this repository is updated seldom

Sorry for the delay, @AndersDJohnson! Yeah, I work on this in my spare time in between other projects, so it can take awhile.

**Comment by AndersDJohnson** [comment]
@btmills Will you be releasing a new version, even a beta one, to include this and other changes? Thanks!

**Pull request merged**